bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx101-vpn-mobile-free-en.html

```diff
@@ -0,0 +1,45 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('protocol-emphasis-box') }}
+  {{ css_bundle('firefox_whatsnew_101_vpn_mobile') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% set play_store_link = play_store_url('firefox') + '&utm_source=WNP&utm_campaign=WNP101' %}
+{% set app_store_link = app_store_url('firefox') %}
```

Comment:
Suggestion (non-blocking): these don't seem to be used anywhere so can we remove them?